schemas/api/heartbeatAcknowledgment.schema.json

```diff
@@ -0,0 +1,47 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
Seems weird that we have two timestamps in the same message:

```
 this.messaging.post({

            type: "heartbeatAcknowledgementRequest",
            meta: {
                requestUuid: this.messaging.createUUID(),
                timestamp: new Date()
            },
            payload: {
                timestamp: new Date()
            }

        } as HeartbeatAcknowledgementRequest)
```

Reply:
The one in the payload is supposed to be a quote of the one in the original event (as described in the description of the field - you should not be using `new Date()` for it). I can switch it to the eventUuid which is probably the better reference to use anyway.

Reply:
There are two timestamps in the heartbeat event as well (one in the payload). I'll lose that as well, seems pointless.

Reply:
Resolved in f4b44a16927e097120beb598e03f08590d905f3f - now uses the eventUuid of the heartbeatEvent and cuts out the extra timestamp fields.